Enhancement: Update Ornatrix Docs (#237)
ynput/ayon-documentation: website/docs/addon_maya_ornatrix_artist.md

AYON supports Ephere's **Ornatrix** with the workflow of publishing and loading maya scene with Ornatrix data, Ornatrix groom presets and Ornatrix Alembic. It also supports to connect your target geometry with the loaded Ornatrix data (either from the maya scene or the groom preset.)

## Workflow
AYON has two creators for Ornatrix publishing: `Ornatrix Rig` and `Ornatrix Cache`.

> ## Ornatrix Rig

Author: I've a question about Ornatrix rig. 

So, this section doesn't mention anything about connecting rigs to pointcaches or animations. 
and I can't understand why would artists need to connect them. 

Could we elaborate on this workflow?

Author: Also, test notes in this PR https://github.com/ynput/ayon-maya/pull/1 look useful. could we use bits of them to write about the workflow and what artists should do and expect ?

Reviewer: > I've a question about Ornatrix rigs.
> 
> So, this section doesn't mention anything about connecting rigs to pointcaches or animations. and I can't understand why would artists need to connect them.
> 
> Could we elaborate on this workflow?

it is similar to how the connect yeti rig works. When loading the animated characters and they are not at the original positions(e.g. they are sitting or running), the connect action can help the rigs to find the correct position of where the fur(e.g. hair) located.

Author: so, we link the animated character with its animated hair/fur/groom to be able to see the full animation.

Author: Thank you for your explanation. it helped me update the answer.
https://github.com/ynput/ayon-documentation/pull/237/commits/f218f5198a3ebd2a14154c0e1a5f6020bbbd50b0

### Loading

The **Ornatrix Cache** supports to load Ornatrix alembic (.abc), Vray Proxy (.abc) and Arnold Scene Source (.abc).

## FAQ
### How to connect an Ornatrix rig with a pointcache or animation?

> After loading the Ornatrix Rig and a pointcache to connect it to, you can use the AYON tools to connect it for you:

Reviewer: ```suggestion
Once you’ve loaded the Ornatrix Rig and its corresponding pointcache, you can connect them using the AYON tools: 
```

Author: what about 
```suggestion
Once you’ve loaded the Ornatrix Rig and its corresponding pointcache, you can connect them using AYON Inventory Actions: 
```

Reviewer: We should use you have instead of you've.

Author: what about: 
```suggestion
Once you load the Ornatrix Rig along with its corresponding pointcache, you can link them using AYON Inventory Actions:
```

Author: I've applied the last suggestion. feel free to unresolve.
https://github.com/ynput/ayon-documentation/pull/237/commits/5df28b622d2dbf737f57502e6b492bda2b003790

1. Go to `AYON` -> `Manage...`
2. In the inventory select the assets from oxrig and animation/pointcache product type.
3. Right-click `Actions` -> `Connect Ornatrix Rig`
The Ornatrix geometry will be connected the animation rig with the transform and mesh attributes.

:::tip

> The connecting will occur by node name - so if it fails, make sure the node name for the geometry is the same across your animation caches and the ornatrix exports.

Reviewer: ```suggestion
The connection depends on the name of node. If it fails, you need to make sure the node name for the geometry is consistent across your animation caches and the Ornatrix exports.
```

Author: what about: 
```suggestion
The connection relies on the node's name. If it fails, ensure that the geometry's node name is consistent between your animation caches and the Ornatrix exports.
```

Author: I've applied the last suggestion. feel free to unresolve.
https://github.com/ynput/ayon-documentation/pull/237/commits/5df28b622d2dbf737f57502e6b492bda2b003790

> **Double geometry:** Be aware that both the ornatrix rig and your animated cache have the mesh geometry. After connecting the fur you will most likely want to hide one of the two to ensure you do not have two overlapping meshes during rendering.

Reviewer: ```suggestion
**Double geometry:** Be aware that your ornatrix rig and animated cache have the mesh geometry. After connecting the fur you will most likely want to hide one of the two to ensure you do not have two overlapping meshes during rendering.
```

### Loading

The **Ornatrix Cache** supports to load Ornatrix alembic (.abc), Vray Proxy (.abc) and Arnold Scene Source (.abc).

## FAQ
### How to connect an Ornatrix rig to a pointcache or animation?

> To guide Ornatrix rig to its proper location. you'll need to connect it to the animation, aligning both transform and mesh attributes.

Reviewer: ```suggestion
To position the Ornatrix rig correctly, you need to connect it to the animation, aligning both the transform and mesh attributes
```

Author: fixed in https://github.com/ynput/ayon-documentation/pull/237/commits/7a84098e9fe5327cfc62278b0b720399bf206088
